packages/itwinui-css/src/alert/alert.scss

```diff
@@ -74,7 +74,7 @@
     @include iui-anchor-status($statusColor);
   }
 
-  #{$rootSelector}-button {
+  #{$rootSelector}-button:focus-visible {
```

Comment:
I don't like that `:focus-visible` is needed here but not needed in [`/toast/categories.scss` line 22](https://github.com/iTwin/iTwinUI/blob/f384be3406512a11b486c583aca14bb427779138/packages/itwinui-css/src/toast/categories.scss#L23).

Reply:
You can get rid of `:focus-visible` if you move things around in `alert/classes.scss`. The alert-button and categories have the same specificity so categories should come after the button if you want them to take preference.
![](https://user-images.githubusercontent.com/9084735/199571866-51b5db1b-f435-402d-bc5f-99bd33dc9454.png)

Reply:
Change made & `:focus-visible` removed, good suggestion.